<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chatWidget</class>
 <widget class="QWidget" name="chatWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>chatWidget</string>
  </property>
  <widget class="QTextEdit" name="textEditreceive">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>431</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>429</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButtonsmile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtontxtsize">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtontelephone">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonfile">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QTextEdit" name="textEditsend">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>350</y>
     <width>431</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>531</x>
     <y>71</y>
     <width>191</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string>对方网络信息</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>290</y>
     <width>161</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string>我方网络信息</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelUserImg">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/image/Luffy.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>510</y>
     <width>311</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButtonclose">
      <property name="text">
       <string>关闭</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonsend">
      <property name="text">
       <string>发送</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_username">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>44</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
